aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/exceptions/handlers/PluginExceptionHandlers.java

```diff
@@ -27,6 +27,7 @@
  */
 public class PluginExceptionHandlers {
     private static final String ALL_EXCEPTIONS = "all";
+    private static final String ALL_EXCEPTIONS_1 = "*";
```

Comment:
Uh, this name is not that pretty) Please name this like `EXCEPTIONS_WILDCARD`. You will reuse it in SelectiveExceptionHandler line 59